docs/src/man/basics.md

```diff
@@ -0,0 +1,1505 @@
+   1 │     0     67  male        2  own      NA               little           ⋯
+                                                               3 columns omitted
+
+julia> last(german)
+DataFrameRow
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+ 1000 │   999     27  male        2  own      moderate         moderate        ⋯
+                                                               4 columns omitted
+```
+
+## Taking a Subset of `german` data frame
+
+### Indexing Syntax
+
+Specific subsets of a data frame can be extracted using the indexing syntax, similar to matrices.
+In the [Indexing](https://dataframes.juliadata.org/stable/lib/indexing/#Indexing) section of the
+manual you can find all details about the available options. Here we highlight the basic ones.
+
+The colon `:` indicates that all items (rows or columns depending on its position) should be retained:
+
+```jldoctest dataframe
+julia> german[1:5, :]
+5×10 DataFrame
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │     0     67  male        2  own      NA               little           ⋯
+   2 │     1     22  female      2  own      little           moderate
+   3 │     2     49  male        1  own      little           NA
+   4 │     3     45  male        2  free     little           little
+   5 │     4     53  male        2  free     little           little           ⋯
+                                                               3 columns omitted
+
+julia> german[[1, 6, 15], :]
+3×10 DataFrame
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │     0     67  male        2  own      NA               little           ⋯
+   2 │     5     35  male        1  free     NA               NA
+   3 │    14     28  female      2  rent     little           little
+                                                               3 columns omitted
+
+julia> german[:, [:Age, :Sex]]
+1000×2 DataFrame
+  Row │ Age    Sex
+      │ Int64  String
+──────┼───────────────
+    1 │    67  male
+    2 │    22  female
+    3 │    49  male
+    4 │    45  male
+    5 │    53  male
+    6 │    35  male
+    7 │    53  male
+    8 │    35  male
+  ⋮   │   ⋮      ⋮
+  994 │    30  male
+  995 │    50  male
+  996 │    31  female
+  997 │    40  male
+  998 │    38  male
+  999 │    23  male
+ 1000 │    27  male
+      985 rows omitted
+
+julia> german[1:5, [:Sex, :Age]]
```

Comment:
this is not an example of using `:`

Reply:
Yeah, I had put the comment outside the block. So that considered for all but now I have removed that example from that block.

Reply:
OK

Reply:
upon checking - you are still in the same logical block - there is no comment between blocks.